keep-alive/src/main/java/KeepAliveGatt.kt

```diff
@@ -142,8 +173,14 @@ class KeepAliveGatt internal constructor(
         isRunning.compareAndSet(false, true) || return false
 
         scope.launch(CoroutineName("KeepAliveGatt@$bluetoothDevice")) {
+            var connectionAttempts = 1
```

Comment:
Documentation describes `connectionAttempt` spanning the lifespan of `KeepAliveGatt`, but this is a local variable to `connect`. It will reset if `disconnect` is called, then subsequently `connect` again.

Perhaps it was desired to have `connectionAttempts` be a property instead of a local variable? Or is resetting after a requested disconnect then connect the desired behavior?

Reply:
Did some thinking on this - I do believe it was an oversight and it should be a property. I've edited it as such and retested the behavior of the KeepAliveGatt.